bin/aurbuild

```diff
@@ -114,6 +114,11 @@ else
     mkpkg_args+=("${@:$OPTIND}")
 fi
 
+if [[ $sign == --sign ]]; then
+    # Pass socket and environment to the build chroot.
+    mkchr_args+=("-d" "$HOME/.gnupg/S.gpg-agent:/build/.gnupg/S.gpg-agent")
```

Comment:
This makes the same mistake as makechrootpkg, in that it breaks when `GNUPGHOME` is set to a non-standard location. Also, `/build` ?